src/work-allocation/release2/components/available-tasks-filter/available-tasks-filter.component.scss

```diff
@@ -0,0 +1,17 @@
+:host ::ng-deep .available-tasks_location-filter-v2 {
```

Comment:
This is identical to the original (apart from the `-v2` suffix. Why is it needed?

Reply:
agree with PaulG. It has to be r2 as decided